content/src/service/validations/validations/ValidationsV4.ts

```diff
@@ -4,43 +4,53 @@ import { ValidationsForContext } from '../Validator'
 
 export const VALIDATIONS_V4: ValidationsForContext = {
```

Comment:
To support deploying V4 entities we need to be sure that all validations are done, if not we may have invalid v4 entities as not all validations were implemented